boa_interner/Cargo.toml

```diff
@@ -15,3 +15,7 @@ serde = { version = "1.0.145", features = ["derive"], optional = true }
 phf = { version = "0.11.1", features = ["macros"] }
 rustc-hash = "1.1.0"
 static_assertions = "1.1.0"
+once_cell = "1.12.1"
+indexmap = "1.9.1"
+utf16_lit = "2.0.2"
```

Comment:
Is this different to our `utf16!()` macro?

Reply:
Oops, forgot to replace this 😅